src/khoj/database/models/__init__.py

```diff
@@ -160,7 +160,7 @@ class ModelType(models.TextChoices):
 
     max_prompt_size = models.IntegerField(default=None, null=True, blank=True)
     tokenizer = models.CharField(max_length=200, default=None, null=True, blank=True)
-    chat_model = models.CharField(max_length=200, default="mistral-7b-instruct-v0.1.Q4_0.gguf")
+    chat_model = models.CharField(max_length=200, default="NousResearch/Hermes-2-Pro-Mistral-7B-GGUF")
```

Comment:
For folks who are using the old model (`mistral-7b-instruct-v0.1.Q4_0.gguf`), will the offline code path still work?

Reply:
The migration script should help folks using the previous default model, rest of the folks will need to manually update/reset the chat model they are using